src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,11 +2,16 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.info("Login was called");
+
         User user = findByLogin(login);
         if (!user.getPassword().equals(password)) {
```

Comment:
The password comparison uses `equals`, which is correct for comparing strings. However, ensure that passwords are handled securely, possibly using hashed values instead of plain text, to enhance security.

Reply:
This is not my code, I should add only logger